src/app/exercise/services/deps-order.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from '@angular/core';
+import { FileConfig } from '../interfaces/file-config';
+
+import * as ts from 'typescript';
+import { simpleVisitor } from './visitor';
+
+
+@Injectable()
+export class DepsService {
+
+  /**
+   * Takes a file path, and another path relative to the first one.
+   * Returns the fulle
+   *
+   * @param path
+   * @param file
+   * @returns {string}
```

Comment:
I don't think you need to provide this kind of JSDoc with typescript since the type is inferred.